kpatch-build/kpatch-elf.c

```diff
@@ -770,8 +771,7 @@ void kpatch_write_output_elf(struct kpatch_elf *kelf, Elf *elf, char *outfile)
 	Elf_Data *data;
 	GElf_Shdr sh;
 
-	/* TODO make this argv */
```

Comment:
I'm not sure I understand this TODO. Make what argv? What this to actually remind to make the change you just did? (pass the permission as argument)

Reply:
I'm not really sure what Jessica meant when adding that TODO comment.  My initial guess was this change, but maybe she thought about plumbing it all the way out to the program `argv[]`?  That seems like overkill to me, so I landed on this PR.

Reply:
Yes, there's no need to make it into a command line option. So, I guess removing that TODO makes sense.